sdks/python/scripts/generate_pydoc.sh

```diff
@@ -174,6 +174,7 @@ nitpicky = True
 nitpick_ignore = []
 nitpick_ignore += [('py:class', iden) for iden in ignore_identifiers]
 nitpick_ignore += [('py:obj', iden) for iden in ignore_identifiers]
+nitpick_ignore += [('py:exc', 'ValueError')]
```

Comment:
Do you know why?

Reply:
I found this issue in Sphinx. https://github.com/sphinx-doc/sphinx/issues/1034